docs/admin/system-information.rst

```diff
@@ -1729,6 +1729,13 @@ After you reindexed all tables, this cluster check will pass.
 License check
 .............
 
+
+.. NOTE::
+
+   This check was removed in CrateDB version 4.5 because it no longer requires
+   a license.
```

Comment:
"it" here is an indeterminate reference. this would be better phrased as: "This check was removed in version 4.5 because CrateDB no longer requires an enterprise license."

Reply:
done :+1: